closure/templates/test/BUILD

```diff
@@ -69,3 +71,16 @@ closure_js_test(
     ],
     language = "ECMASCRIPT6_STRICT",
 )
+
+closure_template_java_library(
```

Comment:
Done, although that didn't seem to work for me even after pulling in the latest commit, and it looks like travis just failed?